docker/nginx/dist/conf/tpotweb.conf

```diff
@@ -46,7 +46,11 @@ server {
     client_body_buffer_size  128k;
     client_header_buffer_size 1k;
     client_max_body_size 2M;
-    large_client_header_buffers 2 1k;
+    
```

Comment:
Please adjust the comment with regard to support reverse proxy chaining.

Reply:
> Please adjust the comment with regard to support reverse proxy chaining.

I have changed the comment to describe why the change is applied and why we are not following OWASP recommendations.

If you think i need to change something or write it differently, just let me know.